regression/pages/studio/terms_of_service.py

```diff
@@ -0,0 +1,15 @@
+"""
+Terms of Service page
+"""
+from bok_choy.page_object import PageObject
+from regression.pages.studio import BASE_URL
+
+
+class TermsOfService(PageObject):
+    """
+    Terms of Service page
+    """
+    url = BASE_URL + '/edx-terms-service'
+
+    def is_browser_on_page(self):
+        return 'edX Terms of Service' in self.browser.title
```

Comment:
I suggest the same thing here...which is to use whatever CSS element contains the terms of service text, and set it to be visible.